Doc/tutorial/controlflow.rst

```diff
@@ -4,8 +4,8 @@
 More Control Flow Tools
 ***********************
 
-Besides the :keyword:`while` statement just introduced, Python knows the usual
-control flow statements known from other languages, with some twists.
+Besides the :keyword:`while` statement just introduced, Python uses the usual 
```

Comment:
Travis is failing because of the trailing whitespace on this line. Please remove it.

Reply:
It's already. Sorry for that, it's my first contribution